web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
I'm not sure what you mean here.

Every manual test in this directory calls `setupManualShareTest` with the expected outputs (i.e., the expected values to be received by the target), which displays those expectations for the user. In this case, the expectation is that the url field contains `document.baseURI`.

You can see this page here:
https://w3c-test.org/submissions/6287/web-share/share-url-empty-manual.html

> **Instructions:**
>
> 1. Click the Share button.
> 2. Choose a valid share target.
>
> **Pass the test iff the target app received:**
>
> * title = ""
> * text = ""
> * url = "https://w3c-test.org/submissions/6287/web-share/share-url-empty-manual.html"

I think that's sufficient indication of what the manual pass condition is, in each test.